src/GAP.jl

```diff
@@ -286,6 +286,9 @@ function __init__()
         push!(cmdline_options, "-b")
     end
 
+    if haskey(ENV, "GAP_BARE_DEPS")
+        push!(cmdline_options, "-A")
```

Comment:
I had expected that `push!(cmdline_options, "--bare")` would be enough, that is, the change in `gap/systemfile.g` would not be necessary. However, apparently then loading JuliaInterface causes problems. Thus the proposed changes seem to be a good solution for the moment.

Reply:
Yeah I first tried `--bare`, too, but then it didn't load JuliaInterface either. Hence this solution now. (Besides, we still might want to always load e.g. GAPDoc... well, we definitely will want to load it and also smallgrp, transgrp, primgrap and a bunch more in Oscar.)